tests/test_server.py

```diff
@@ -1323,3 +1323,10 @@ async def test_resource_deploy_done_error_handling(server, client, environment,
         change=const.Change.nochange,
     )
     assert result.code == 404, result.result
+
+
+@pytest.mark.asyncio
+async def test_start_location_no_redirect(server):
```

Comment:
In general, test cases should have a docstring with a short description of what the test case is testing. Experience has shown it's in many cases not easy to extract that information from the code alone. Could you add a docstring here?